### DISCUSSION TITLE
Drawing using vectors so my app is resolution independent

### DISCUSSION THREAD

**Discussion by ghost** [spritewidget/spritewidget#80] [Q&A]
I am building an app to run on small and large screens.
Does this package allow the "Sprite" to be described as vector and be rendered at runtime to different resolution.

Or maybe I should use the built in paint API in flutter ?

**Comment by vlidholt** [comment]
@gedw99 Yes, you can do vector drawing inside the SpriteWidget. Basically, you just use the Flutter `Canvas`.

This is a minimal example of creating a node with custom drawing:
```
class RedCircle extends Node {
  RedCircle(this.radius);

  double radius;

  @override
  void paint(Canvas canvas) {
    canvas.drawCircle(
      Offset.zero,
      radius,
      new Paint()..color = const Color(0xffff0000)
    );
  }
}
```
